### DISCUSSION TITLE
Project still in development?

### DISCUSSION THREAD

**Discussion by tvb** [okaybenji/text-engine#29] [General]
Hi @okaybenji ,

I was wondering if you still are maintaining and developing this project. From all the projects for text-based engines this one is by far the best. Would be a real shame if the project is dead.

**Comment by okaybenji** [comment]
Hey, thank you very much! I am still around and I still care a lot about this engine. There isn't anything I'm really itching to add at this point but I'm happy to answer questions or fix any bugs that might be uncovered. I really hope people continue to enjoy the engine. I love to see the games folks make with it.

**Comment by tvb** [comment]
@okaybenji That is great to hear! We'll be working on implementing this engine in one of our projects and hopefully we can provide you some feedback soon. Let's keep in touch 👍🏻

**Comment by gntsketches** [comment]
@okaybenji Just wanted to chime in here. I haven't had time to engage with this project recently but I think it's terrific and hope to do more before long. Glad to hear it's still being taken care of.

**Reply by okaybenji** [reply]
🙏 Appreciate the feedback. Please do share if you make something!
